packages/blazor-workspace/Tests/SprightBlazor.Tests.Acceptance/Pages/Rectangle.razor

```diff
@@ -1,6 +1,7 @@
 ﻿@page "/Rectangle"
 @namespace SprightBlazor.Tests.Acceptance.Pages
 @inherits LayoutComponentBase
+@rendermode @(new InteractiveServerRenderMode(prerender: false))
```

Comment:
Having prerendering on for acceptance tests, for an interactive render mode, can cause issues because the page/components appear ready for interaction, but without event handlers hooked up, when prerendered. The docs don't make it super clear when the transition to fully interactive components will happen, but I haven't seen the test failures since I disabled prerendering on the test pages.

So basically, prerendering disabled + waiting for our JS initializers to report ready => Playwright can definitely interact with the page at that point (combined with Playwright's existing actionability checks like waiting for the element to be visible and non-zero size).

I did update the docs on creating tests to specify `prerender: false`, I'll see if there's anything else I should call out there though.

Reply:
Also added a sentence in the acceptance test section stating that prerendering should be disabled (and why).